app/controllers/web/sessions_controller.rb

```diff
@@ -23,4 +24,26 @@ def create
       end
     end
   end
+
+  def validate_otp
+    signin_params = params
```

Comment:
That same suggestion here.

Reply:
Done! 2f7fa38ae843a061dae32cb3634c374892fcb37a